src/components/notifications/Notifications.tsx

```diff
@@ -106,11 +110,21 @@ const Notifications = ({
   }, [webRef])
 
   const { width } = Dimensions.get('window')
-  const initialPosition = -1 * width + INITIAL_OFFSET
+
+  // When a navigation stack is open, don't allow the notifications to be opened
```

Comment:
this seems find, but is a bit wonky (the initial way i set it up was wonky). I wonder if there is a more canonical navigation stack solution so that the left swipe gesture is *fully* supported rather than there just being this slight border we can latch onto

Reply:
I think we could probably use react gesture handler or something to catch a swipe from the left edge